pom.xml

```diff
@@ -1333,27 +1335,60 @@
           <plugin>
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.5</version>
+            <version>${jacoco.version}</version>
+            <configuration>
+              <!--
+                  Append all coverage data into a single .exec file, so the reports include
+                  integration test results. The report generation should happen in a separate,
+                  later stage, so the coverage data is complete with the results of all the tests
+                  in the project.
+               -->
+              <append>true</append>
+              <destFile>${build.path}/../target/jacoco-combined.exec</destFile>
+            </configuration>
             <executions>
               <execution>
                 <!-- offline instrumentation is required, for jacoco to work with powermock tests -->
-                <id>jacoco-instrument-classes</id>
+                <id>jacoco-unit-instrument-classes</id>
```

Comment:
I couldn't find this id in the jacoco documentation. Do you have a pointer to it?

Reply:
the `id` is something we can make up. It is the name of the execution.

The `goal` is the name of the specific goal of each plugin that is already predefined.